### ISSUE TITLE
Attention to detail

### ISSUE THREAD

**Issue by aldoborrero** [EthVM/ethvm#318]
For some major components:
- [x] PageBlocks
- [x] PageTxs
- [x] PagePendingTxs
- [x] PageHome
- [x] PageDetailsBlock. See #349
- [x] PageDetailsUncle. See #352 
- [x] PageUncles. See #356 
- [x] PageDetailsTx. See #345
- [x] PageDetailsToken
- [x] PageDetailsAddress. See #358 

Also we need to add:

- [x] TableTxs & TableTxsRow: First column display Block # and link to /block/{hash}. (See: https://etherscan.io/txs)
- [x] TableUncles & TableUnclesRow: After block height, Uncle Number (see: https://etherscan.io/uncles)

After everything is done. We need to double check that:

- [x] AppError.vue (not to be confused with AppError2.vue) has been removed and we are only using AppError2.vue renamed as AppError.vue

**Comment by gsiuniak** [comment]
Made/edited standard AppError component. Added to tokens/transactions. Need to reformat other pages to work properly.

(Also created/implemented AppLoading component, and implemented in same views)

**Comment by aldoborrero** [comment]
@gsiuniak perfect, whenever this feature is ready to be merged on develop, go ahead and do it directly. We can create other mini tasks, related to attention to "detail".

**Comment by gsiuniak** [comment]
Note: For TableUncles & TableUnclesRow, I added Uncle Number. However, on Uncle Details, the Block Height/Uncle Number are switched. So one of the two views is incorrect. I attached screenshots showing the discrepancy. I can easily change to the correct info.

![screenshot from 2019-02-19 17-43-32](https://user-images.githubusercontent.com/3878092/53060392-37947080-346f-11e9-970e-a0fd30bcd44e.png)
![screenshot from 2019-02-19 17-43-51](https://user-images.githubusercontent.com/3878092/53060396-39f6ca80-346f-11e9-9e4c-f5ec69a64d7d.png)
